doc/osmp_test_architecture.md

```diff
@@ -0,0 +1,224 @@
+# OSMP Test Architecture
+
+The OSMP Test Architecture is designed to test any simulation model complying to the [OSI Sensor Model Packaging (OSMP)](https://github.com/OpenSimulationInterface/osi-sensor-model-packaging) specification.
+Contrary to the S in OSMP, a variety of model typed from environmental effect models over sensor models to traffic participant models can be tested with this architecture.
```

Comment:
I think there was the idea to rename it to "OSI Simulation Model Packaging" - I will ask about that

Reply:
https://github.com/OpenSimulationInterface/osi-sensor-model-packaging/issues/67

Reply:
FYI: I did write Pierre and Ben to put this on the agenda of the OSI Maintainer Group. Until it is renamed we leave this explanation.